phys2bids/tests/test_utils.py

```diff
@@ -0,0 +1,98 @@
+"""
+    Runs unit tests on utils.py
+"""
+
+import os
+from phys2bids import utils
+
+
+# Tests check_input_dir
+def test_check_input_dir():
+    test_path = '/home/mrmeeseeks/blue/'
+    path_corrected = utils.check_input_dir(test_path)
+    assert path_corrected == test_path[:-1]
+
+
+# Tests check_input_ext
+def test_check_input_ext():
+    test_filename = 'help'
+    ext = 'py'
+    out_filename = utils.check_input_ext(test_filename, ext)
+    assert str(out_filename) == f'{test_filename}.{ext}'
+
+
+# Tests check_input_type
+def test_check_input_type():
+    test_filename = 'Test_belt_pulse_samefreq.acq'
+    test_path = '/home/travis/build/physiopy/phys2bids/phys2bids/tests/data'
+    assert utils.check_input_type(test_filename, test_path)
+
+
+# Tests path_exists_or_make_it
+def test_path_exists_or_make_it():
+    test_folder = '/home/travis/build/physiopy/phys2bids/tests/foo'
+    utils.path_exists_or_make_it(test_folder)
+    os.path.isdir(test_folder)
+    os.rmdir(test_folder)  # Removes the created folder
+
+
+# Tests check_file_exists
+def test_check_file_exists():
+    test_filename = 'Test_belt_pulse_samefreq.acq'
+    test_path = '/home/travis/build/physiopy/phys2bids/tests/data'
+    test_full_path = os.path.join(test_path, test_filename)
+    utils.check_file_exists(test_full_path)
+
+
+# Tests move_file
+def test_move_file():
+    ext = '.txt'
+    test_path = '/home/travis/build/physiopy/phys2bids/tests/data'
+    test_old_path = os.path.join(test_path, 'foo.txt')
+    open(test_old_path, 'a').close()
```

Comment:
```suggestion
    with open(test_old_path, 'a'):
        pass
```

might be a bit safer!